pages/testCases/kundur_two_area_system/index.md

```diff
@@ -0,0 +1,475 @@
+| G2             | $$700.000 + \mathrm{j}234.676$$ | $$700 + \mathrm{j}235$$       |
+| G3             | $$719.000 + \mathrm{j}175.986$$ | $$719 + \mathrm{j}176$$       |
+| G4             | $$700.000 + \mathrm{j}202.071$$ | $$700 + \mathrm{j}202$$       |
+
+
+- The active power exchange from Area 1 to Area 2 (see
+  <a href="#fig-two-area-system" class="quarto-xref">Figure 1</a>) is
+  400.431 MW in the Dyna$$\omega$$o implementation and 400 MW in the
+  Kundur reference.
+
+### Example a
+
+*Example a* examines small-signal performance of the system with
+constant excitation.
+
+- All four generators are on manual excitation (field voltage
+  $$E_\mathrm{fd} = \mathrm{const.}$$)
+- Constant current characteristics of active load components
+- Constant impedance characteristics of reactive load components
+
+<a href="#fig-example-a-eigenvalues" class="quarto-xref">Figure 2</a>
+shows the result of an eigenvalue analysis of this system, compared to
+the reference values from [1]. The remaining differences probalby
+originate from differing model implementations of the synchronous
+machines in Dyna$$\omega$$ compared to the ones used by Kundur.
+
+
+![](images/20240814123344.png)
+
+<div id="fig-example-a-eigenvalues">
+
+Figure 2: Eigenvalues of the system with constant excitation voltage
+
+</div>
+
+### Example b
+
+> [!NOTE]
+>
+> To obtain a steady state initial condition, the exact values for the
+> generators active and reactive power and voltage angles from load flow
+> (<a href="#sec-loadflow" class="quarto-xref">Section 6.1</a>) need to
+> be used as initial values in the dynamic simulation.
+
+> [!NOTE]
+>
+> In the Dyna$$\omega$$o implementation, if the voltage transducer time
+> constant $$T_\mathrm{R}$$ = 0.01 s, then the voltage measurement first
+> order lag and the field flux linkage $$\lambda_\mathrm{f}$$ oscillate
+> against each other with high amplitude. This is not represented in the
+> reference results of [2]. To avoid this behaviour, set
+> $$T_\mathrm{R} = 10^{-6}$$ s. This has been done for the Dyna$$\omega$$o
+> implementation.
+
+#### Part 1: Self-excieted dc exciter (DC1A)
+
+Instead of constant excitation voltage as in
+<a href="#sec-exampleA" class="quarto-xref">Section 6.2</a>, now the
+self-excited DC exciter DC1A is used. For parameters, see
+<a href="#tbl-dc1a-parameters" class="quarto-xref">Table 3</a>.
+
+In [2] this case has not been taken into account for comparative
+analysis. Like there, it has not been investigated using the
+Dyna$$\omega$$o implementation.
+
+#### Part 2: Thyristor exciter (ST1A) with high transient gain
+
+Instead of constant excitation voltage as in
+<a href="#sec-exampleA" class="quarto-xref">Section 6.2</a>, now the
+thyristor exciter ST1A is used. For parameters, see
+<a href="#tbl-dc1a-parameters" class="quarto-xref">Table 3</a>. It has
+high transient gain, i.e. no transient gain reduction.
+
+The implementation in Dyna$$\omega$$o does not allow to set saturation
```

Comment:
Is it possible that the saturation is the cause of the difference between the Kundur and Dynawo eigenvalues? How do the other eigenvalues move when trying to match the unstable one?
Instead of matching the unstable eigenvalue perhaps try to match as closely as possible the Kundur and Shackshaft saturation curves?